mne/viz/utils.py

```diff
@@ -1017,6 +1017,7 @@ def plot_sensors(
     sphere=None,
     pointsize=None,
     linewidth=2,
+    cmap=None,
```

Comment:
We've started adding these with any new `kwargs`, so now is a good time to do it
```suggestion
    *,
    cmap=None,
```